### PR TITLE
Enhance and unify documentation

### PR DESCRIPTION
**Pull request by markuspg**
This PR raises the documentation coverage of patir to 100%. Additionally the language of the documentation comments has been unified to generally relying on passive language instead of addressing the reader directly sometimes.

Since future pull requests will change great deals of code (e.g. unifying the formatting) this change has been made before to have the documentation strings as additional hooks to aid in avoiding merge conflicts and improve the readability of diffs.

### CODE REVIEW

**Comment by damphyr** [comment]
By eliminating all the fun informalities you are removing the last vestiges of my youth 😛 
Damn you... and thanks 😄

File: README.md [hunk 1]
```diff
@@ -4,20 +4,25 @@
...
+* a command abstraction with a platform independent implementation for running
+  shell commands or Ruby code
+* sequences of commands using the same command abstraction as single commands.
+* a Configuration class and format for loading configuration files written in Ruby.
 
 ## Why?
-We've been using the same things again and again and can't be bothered to code it anew every time.
 
-The command abstraction has been used the most, the Logger defaults and formatting the least.
+Some same things are used again and again and shouldn't be rewritten every time.
```
Comment (mrkita) [inline README.md]: * Some of the same things are used ...

File: README.md [hunk 2]
```diff
@@ -4,20 +4,25 @@
...
+
+The command abstraction has been used the most, the logger defaults and
+formatting the least.
```
Comment (mrkita) [inline README.md]: Focuses on the command abstraction, logger defaults, and ??

File: README.md (hunk 1 above)
Comment (markuspg) [reply README.md]: I fixed the text as proposed by you :+1:

File: README.md (hunk 2 above)
Comment (markuspg) [reply README.md]: I changed the paragraph as follows to increase clarity:

```
The command abstraction is the primary and most used feature of this gem. The
logger creation convenience method and the adjusted logger formatter are used
the least.
```

**Review by mrkita — APPROVED** [review]

**Pull request merged**